mockup/patterns/relateditems/pattern.js

```diff
@@ -98,11 +98,11 @@ define([
   'mockup-utils',
   'pat-registry',
   'translate',
-  'text!mockup-patterns-relateditems-url/templates/breadcrumb.xml',
-  'text!mockup-patterns-relateditems-url/templates/favorite.xml',
-  'text!mockup-patterns-relateditems-url/templates/result.xml',
-  'text!mockup-patterns-relateditems-url/templates/selection.xml',
-  'text!mockup-patterns-relateditems-url/templates/toolbar.xml',
+//  'text!mockup-patterns-relateditems-url/templates/breadcrumb.xml',
+//  'text!mockup-patterns-relateditems-url/templates/favorite.xml',
+//  'text!mockup-patterns-relateditems-url/templates/result.xml',
+//  'text!mockup-patterns-relateditems-url/templates/selection.xml',
+//  'text!mockup-patterns-relateditems-url/templates/toolbar.xml',
```

Comment:
That's strange. Possibly it is related to other TTW build failures when "++plone++unique" was introduced in late 5.0. That broke relative URLs in build.